src/main/isEmail.ts

```diff
@@ -1,20 +1,60 @@
 import {
   registerDecorator,
+  ValidationArguments,
   ValidationOptions,
   ValidatorConstraint,
-  ValidatorConstraintInterface,
-  ValidationArguments
+  ValidatorConstraintInterface
 } from '@hmcts/class-validator'
-
-import * as validator from 'validator'
+import * as punycode from 'punycode'
 
 @ValidatorConstraint()
 export class IsEmailConstraint implements ValidatorConstraintInterface {
+  private static EMAIL_REGEX = new RegExp('^[a-z0-9.!#$%&\'*+/=?^_`{|}~\\-]+@([^.@][^@\\s]+)$', 'i')
+  private static HOSTNAME_PART_REGEX = new RegExp('^(xn-|[a-z0-9]+)(-[a-z0-9]+)*$', 'i')
+  private static TLD_REGEX = new RegExp('^([a-z]{2,63}|xn--([a-z0-9]+-)*[a-z0-9]+)$', 'i')
```

Comment:
These could use non-capturing groups but for such short expressions and inputs the difference is probably negligible